### PR TITLE
enable templates for Nim

### PR DESCRIPTION
**Pull request by xflywind**

### CODE REVIEW

File: benchmark [hunk 1]
```diff
@@ -339,6 +339,7 @@ def main():
             results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Check', templated=False)
         if 'Build' in args.operations:
             results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Build', templated=False)
+	    results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Build', templated=True)
```
Comment (xflywind) [inline benchmark:342]:
```suggestion
	        results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Build', templated=True)
```

File: benchmark [hunk 2]
```diff
@@ -339,6 +339,7 @@ def main():
             results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Check', templated=False)
         if 'Build' in args.operations:
             results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Build', templated=False)
+	        results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Build', templated=True)
```
Comment (xflywind) [inline benchmark:342]:
```suggestion
	      results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Build', templated=True)
```

File: benchmark [hunk 3]
```diff
@@ -339,6 +339,7 @@ def main():
             results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Check', templated=False)
         if 'Build' in args.operations:
             results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Build', templated=False)
+	      results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Build', templated=True)
```
Comment (timotheecour) [inline benchmark:342]: tab vs space => error

Comment (xflywind) [inline benchmark:342]:
```suggestion
            results += benchmark_Nim(execs=execs, durs=durs, gpaths=gpaths, args=args, op='Build', templated=True)
```

Comment (xflywind) [reply benchmark]: fixed

**Pull request closed without merging**